src/renderer/src/design/icons/IconAdd.tsx

```
@@ -0,0 +1,36 @@
+import { SVGProps, useId } from 'react'
+
+const IconAdd = (props: SVGProps<SVGSVGElement>) => {
```

ESLint

[error] Missing return type on function.

(@typescript-eslint/explicit-function-return-type)